qa/L0_perf_nomodel/test.sh

```diff
@@ -139,7 +141,13 @@ TEST_CONCURRENCY+=(
     1
     16
     16)
-TEST_BACKENDS="plan custom graphdef savedmodel onnx libtorch python"
+
+# Only TF, Onnx and TRT are supported on Jetson
+if [ "$TEST_JETSON" -eq 1 ]; then
+    TEST_BACKENDS="plan custom graphdef savedmodel onnx"
```

Comment:
Do we still have "custom"?

Reply:
Yes. We use the new backend api for the identity / addsub models and refer to these as custom models